_includes/footer.html

```diff
@@ -9,7 +9,7 @@
             <section>
               <!-- replace with uls -->
               <h6>{{ site.data.[page.lang].settings.nav.groups.agencies }}</h6>
-              <a class="font-xs" href="https://partners.login.gov">{{ site.data.[page.lang].settings.nav.become_a_partner }}</a>
+              <a class="font-xs" href="{{ '/partners/' | locale_url }}">{{ site.data.[page.lang].settings.nav.become_a_partner }}</a>
```

Comment:
If we don't have localized versions of the partners page, we could skip the `locale_url` filter (which is what adds the language-specific prefixes like `/fr/` and `/es/`).

```suggestion
              <a class="font-xs" href="{{ '/partners/' | prepend: site.baseurl }}">{{ site.data.[page.lang].settings.nav.become_a_partner }}</a>
```